openlibrary/templates/books/author-autocomplete.html

```diff
@@ -35,7 +35,7 @@
     <div class="input">
         <input name="$name_path--$i" class="author author-autocomplete" type="text" id="author-$i" value="$author.name"/>
         <input name="$dict_path--$i--author--key" type="hidden" id="author-$i-key" value="$author.key" />
-        <a href="javascript:;" class="remove red plain" title="$_('Remove this author')">[&times;]</a>&nbsp;
+        <a href="javascript:;" class="remove red plain" style="visibility: hidden;" title="$_('Remove this author')">[&times;]</a>&nbsp;
```

Comment:
Why an inline style and not the `hidden--nojs` class you suggested?
I'd rather we avoid a style if we can.

Reply:
@jdlrobson  I'm sorry, I thought this is what you were meant when you said: " I think you should be able to add that class to the 'x' element and that's the task done." Ill look into it.